rails_multitenant.gemspec

```diff
@@ -21,8 +21,8 @@ Gem::Specification.new do |spec|
 
   spec.add_development_dependency "bundler", "~> 1.10"
 
-  spec.add_dependency 'activerecord', ENV.fetch('RAILS_VERSION', ['>= 4.1', '< 5.2'])
-  spec.add_dependency 'activesupport', ENV.fetch('RAILS_VERSION', ['>= 4.1', '< 5.2'])
+  spec.add_dependency 'activerecord', ENV.fetch('RAILS_VERSION', ['>= 4.1', '< 6.0'])
```

Comment:
nit: I know the next Rails release is likely to be 6.0 but we typically do `< 5.3` just to be safe.